repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/TestProjectConfigurationUtils.scala

```diff
@@ -0,0 +1,51 @@
+package cromwell.pipeline.datastorage.dao.repository.utils
+
+import java.nio.file.{ Path, Paths }
+
+import cromwell.pipeline.datastorage.dto._
+import org.scalacheck.Gen
+
+object TestProjectConfigurationUtils {
+  lazy val typedValueGen: Gen[TypedValue] =
+    Gen.oneOf(stringTypedGen, fileTypedGen, intTypedGen, floatTypedGen, booleanTypedGen)
+
+  lazy val fileParameterGen: Gen[FileParameter] =
+    for {
+      nameField <- stringGen
+      valueField <- typedValueGen
+    } yield FileParameter(nameField, valueField)
+
+  lazy val projectFileConfigurationGen: Gen[ProjectFileConfiguration] =
+    for {
+      pathField <- fileGen
+      inputsField <- Gen.nonEmptyListOf(fileParameterGen).suchThat(_.length < 4)
```

Comment:
change `nonEmptyListOf(fileParameterGen).suchThat(_.length < 4)` onto `listOfN(4, fileParameterGen)` for example to not get a large array

Reply:
done